js-miniapp-sdk/test/miniapp.spec.ts

```diff
@@ -42,3 +45,27 @@ describe('requestPermission', () => {
     );
   });
 });
+
+describe('showInterstitialAd', () => {
+  it('should delegate to showInterstitialAd function when request to show interstitial ad in native sdk', () => {
+    const spy = sinon.spy(miniApp, 'showInterstitialAd' as any);
+
+    miniApp.showInterstitialAd();
+
+    return expect(spy.callCount).to.equal(1);
```

Comment:
I'm not understanding what this test is for. You're setting a spy on the function that you're testing, so this expectation is not actually testing anything in the real code. This is only testing that you've set a spy on `showInterstitialAd`. 

I think you copied this from `requestPermission` above? In the case of `requestPermission`, there are two methods: `requestPermission` and `requestLocationPermission`, so that's why there was a spy being used on one of the methods. In the case of `showInterstitialAd`, I think we don't need this test.

Reply:
Removed! I was thinking to show that the method is called once!